controllers/imageController.js

```diff
@@ -69,6 +72,26 @@ ImageController.prototype.setUserAvatar = function(req, res, next) {
 	})
 }
 
+
+ImageController.prototype.setUserAvatar = function(req, res, next) {
+	var processor = new ImageProcessor(this._fs)
+	return this._setUserProfileImage(req, res, next,
+		processor.handleUserAvatarUpload.bind(processor),
+		'/' + req.user.username + '/profile',
+		{original: 'avatarOriginal', scaled: 'avatarScaled'}
+	)
+}
+
+ImageController.prototype.setUserHeader = function(req, res, next) {
+	var processor = new ImageProcessor(this._fs)
+	return this._setUserProfileImage(req, res, next,
+		processor.handleUserHeaderUpload.bind(processor),
+		'/' + req.user.username + '/header',
```

Comment:
this isn't interim, in that it puts stuff in the database, and I'd rather not have to move stuff around. let's put it in the right place the first time.